src/main/java/com/agilethought/internship/sso/config/SecurityConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package com.agilethought.internship.sso.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+	@Override
+	public void configure(WebSecurity web) throws Exception {
+		web.ignoring().antMatchers("/api/v1/user");
```

Comment:
This is allowing only the GET operation what about POST and PUT ?
Please add it if there is any

Reply:
POST is using the same endpoint as GET, and there is no PUT method in the code